docs/main/user-guides/local-storage-user-guide/local-pv-lvm/lvm-configuration.md

```diff
@@ -148,11 +148,36 @@ The following matrix shows standard StorageClass parameters for Local PV LVM.
     <td> <a href="#thinprovision-optional"> thinProvision </a></td>
     <td> yes </td>
     <td> Supported </td>
+    <td> Yes </td>
+  </tr>
+  <tr>
+    <td> <a href="#scheduler"> scheduler </a></td>
+    <td> CapacityWeighted or VolumeWeighted </td>
```

Comment:
Even SpaceWeighted needs to be mentioned, right?

Reply:
User need not use `scheduler` parameter to select SpaceWeighted as its default..